src/main/resources/hudson/plugins/gradle/Gradle/config.jelly

```diff
@@ -16,8 +16,8 @@
         <f:entry title="${%Make gradlew executable}" field="makeExecutable">
             <f:checkbox />
         </f:entry>
-        <f:entry title="${%From Root Build Script Dir}" field="fromRootBuildScriptDir">
-            <f:checkbox default="true"/>
+        <f:entry title="${%Wrapper location}" field="wrapperLocation"> <!-- Optional -->
```

Comment:
When I configure the job to use a wrapper it's not entirely clear where the wrapper comes from.  It would be nice if the plugin explained where it searches for the wrapper and that providing the location is only needed if the wrapper is not in one of the default locations.